docs/markdown/Guidelines.md

```diff
@@ -593,36 +603,38 @@
         <p>You could also acknowledge the user's interaction by showing or deleting the element directly. In this case provide visual feedback such as highlighting the new element or making the element disappear. </p>
         <br />
         <table class="table techne-table">
-            <tbody>
+            <thead>
                 <tr>
                     <th></th>
                     <th>Notifications</th>
                     <th>Errors</th>
                     <th>Success/Confirmation</th>
                 </tr>
+            </thead>
+            <tbody>
                 <tr>
-                    <td>Background color</td>
-                    <td>Dark blue</td>
-                    <td>Red</td>
-                    <td>Dark blue</td>
+                    <th>Background color</th>
+                    <td data-th="Notifications">Dark blue</td>
+                    <td data-th="Errros">Red</td>
+                    <td data-th="Success/Confirmation">Dark blue</td>
                 </tr>
                 <tr>
-                    <td>Display time</td>
-                    <td>3 seconds before they  disappear on their own.</td>
-                    <td>Must be dismissed by clicking or tapping X.</td>
-                    <td>3 seconds before they  disappear on their own.</td>
+                    <th>Display time</th>
+                    <td data-th="Notifications">3 seconds before they  disappear on their own.</td>
+                    <td data-th="Errros">Must be dismissed by clicking or tapping X.</td>
+                    <td data-th="Success/Confirmation">3 seconds before they  disappear on their own.</td>
                 </tr>
                 <tr>
-                    <td>Keyword</td>
+                    <th>Keyword</th>
                     <td></td>
-                    <td>Always 'unsuccessful'</td>
-                    <td>Use action i.e. 'saved,' 'added,' 'sent,' etc. If no direct action is available, use 'successful'.</td>
+                    <td data-th="Errros">Always 'unsuccessful'</td>
+                    <td data-th="Success/Confirmation">Use action i.e. 'saved,' 'added,' 'sent,' etc. If no direct action is available, use 'successful'.</td>
                 </tr>
                 <tr>
-                    <td>Alternative</td>
+                    <th>Alternative</th>
```

Comment:
this should be a td instead of a th.  IT's breaking the display of the table in mobile